timeeval/adapters/docker.py

```diff
@@ -67,6 +67,33 @@ def to_json_string(self) -> str:
 
 
 class DockerAdapter(Adapter):
+    """
+    An adapter that allows to run a Docker image as an anomaly detector.
+
+    Parameters
+    ----------
+
+    image_name : str
+        The name of the Docker image to run.
+
+    tag : str
+        The tag of the Docker image to run. Defaults to "latest".
+
+    group_privileges : str
+        The group privileges to use for the Docker container. Defaults to "akita".
+
+    skip_pull : bool
+        Whether to skip pulling the Docker image. Defaults to False.
+
+    timeout : Optional[Duration]
+        The timeout for the Docker container. Defaults to None.
+
+    memory_limit_overwrite : Optional[int]
+        The memory limit for the Docker container. Defaults to None.
+
+    cpu_limit_overwrite : Optional[float]
+        The CPU limit for the Docker container. Defaults to None.
```

Comment:
Please describe that if they are not set, the TimeEval-timeouts from the supplied `ResourceConstraints` are used. The user should not need to use these arguments.